### ISSUE TITLE
`Pick` and `Omit`

### ISSUE THREAD

**Issue by reinismu** [badrap/valita#6]
This could be a nice addition to objects. There are cases when we need to take only a part of the object.

```
cosnt TestObject = v.object({
    id: v.string(),
    someValue: v.string().asserts(...)
});

const input = TestObject.pick(['someValue']);
```

What are your thoughts on this?

**Comment by jviide** [comment]
I've been thinking about this. One caveat that comes to mind that `.pick` and `.omit` should not probably keep the `.check` checks, as e.g. omitting a key could break `.check`s that assume that the key exist in the type. What do you think?

**Comment by reinismu** [comment]
Yup after pick/omit check should be removed

**Comment by jviide** [comment]
Published v0.0.17 that contains `v.object(...).pick("a", "b")` and `v.object(...).omit("a", "b")` 🙂 Thanks for the suggestion!
